probes/securityPolicyContainsVulnerabilityDisclosure/impl.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package securityPolicyContainsVulnerabilityDisclosure
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/internal/utils"
+)
+
+//go:embed *.yml
+var fs embed.FS
+
+const Probe = "securityPolicyContainsVulnerabilityDisclosure"
+
+func matches(file checker.File) bool {
+	return file.Type != finding.FileTypeURL
+}
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	if raw == nil {
+		return nil, "", fmt.Errorf("%w: raw", utils.ErrNil)
+	}
+	var findings []finding.Finding
+	policies := raw.SecurityPolicyResults.PolicyFiles
+	for i := range policies {
+		policy := &policies[i]
+		if !matches(policy.File) {
+			continue
```

Comment:
```
// FileTypeURL is used in Security-Policy to
// only denote for the details report that the
// policy was found at the org level rather
// than the repo level
```

The other probes don't check for this. This extra matching seems like an unintended scoring change.

Reply:
Removed, thanks